Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -363,9 +363,7 @@ - (BOOL)canAdjustText {
 }
 
 - (BOOL)canFindInPage {
-    // The presentedViewController check is to prevent the find-in-page icon from responding if
-    // the text size adjustment slider is onscreen. Appears to be a UIKit bug.
-    return self.article != nil && (self.presentedViewController == nil);
```

Comment:
@coreyfloyd it's still an issue but I moved the check so that it happens after the button is tapped. that fixes the button state issue and matches how the other buttons behave.